public/pages/Dashboard/containers/Dashboard.js

```diff
@@ -102,20 +104,37 @@ export default class Dashboard extends Component {
     getIsCommentsEnabled(this.props.httpClient).then((commentsEnabled) => {
       this.setState({ commentsEnabled });
     });
+    this.getUpdatedAgentConfig();
   }
 
   componentDidUpdate(prevProps, prevState) {
     const prevQuery = getQueryObjectFromState(prevState);
     const currQuery = getQueryObjectFromState(this.state);
     if (!_.isEqual(prevQuery, currQuery)) {
+      this.getUpdatedAgentConfig();
       this.getUpdatedAlerts();
     }
     if (isDataSourceChanged(prevProps, this.props)) {
       this.dataSourceQuery = getDataSourceQueryObj();
+      this.getUpdatedAgentConfig();
       this.getUpdatedAlerts();
     }
   }
 
+  getUpdatedAgentConfig() {
+    const dataSourceId = getDataSourceId();
+    console.log("ds", dataSourceId);
+    if (dataSourceId){
+      getIsAgentConfigured(dataSourceId).then((isAgentConfigured) => {
+        this.setState({ isAgentConfigured });
+        console.log("UpAC", this.state.isAgentConfigured);
+      });
+    }
+    else{
+      this.setState({ isAgentConfigured: false });
+    }
```

Comment:
I think local data cluster has empty dataSourceId "". But we can also check its agent configuration. Can you check if it will be always false for local cluster?

Reply:
Yes, the local cluster should be checked either, I've updated to check it.